pkg/image/utilities/utilities.go

```diff
@@ -565,7 +565,6 @@ docker save $(echo ${pulled}) | gzip --stdout > ${images}
 $script_name = $MyInvocation.InvocationName
 $image_list = "rancher-windows-images.txt"
 $images = "rancher-windows-images.tar.gz"
-$os_release_id = $(Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\' | Select-Object -ExpandProperty ReleaseId)
 
 $registry = $null
 $help = $false
```

Comment:
we should remove like 577 and 600-602 as the option won't have any effect

Reply:
Right, that makes sense. I'll update.